library/general/src/lib/y2issues.rb

```diff
@@ -46,6 +46,7 @@ def self.report(issues)
 require "y2issues/presenter"
 require "y2issues/location"
 require "y2issues/reporter"
+require "y2issues/autoinst_reporter"
```

Comment:
@imobachgs I was thinking... should we also add a `Y2Issues.autoinst_report` analogous to the `Y2Issues.report` defined a few lines above?

Or maybe add a parameter to it to specify an alternative reporter (not sure exactly in which form)?